web/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    'env': {
+        'browser': true,
+        'es2021': true,
+    },
+    'extends': [
+        // "react-app",
```

Comment:
Nit: do we need these lines?

Reply:
we don't. And I can remove them. though we will want them all if we go the react route. I figured rather than making us hunt too hard they'd be good in the history. I'll remove them now that they are ;)